dashboard/app/views/layouts/_header.html.haml

```diff
@@ -77,9 +77,9 @@
                 %i.fa.fa-pencil
               %a.header_status_bar.level_free_play.applab_free_play{title: t('nav.header.free_play.applab'), href: applab_project_projects_path, style: 'display: none;'}
                 %i.fa.fa-mobile
-            - if params[:controller] !~ /^sessions/ and signin_button_enabled
+            - if !@hide_sign_in_option and signin_button_enabled
```

Comment:
nit: Prefer `&&` over `and`, see our [styleguide](https://github.com/bbatsov/ruby-style-guide#no-and-or-or).